cypari2/pari_instance.pyx

```diff
@@ -385,22 +387,22 @@ def prec_words_to_dec(long prec_in_words):
 
 # Callbacks from PARI to print stuff using sys.stdout.write() instead
 # of C library functions like puts().
-cdef PariOUT sage_pariOut
+cdef PariOUT python_pariOut
 
-cdef void sage_putchar(char c):
+cdef void python_putchar(char c):
     cdef char s[2]
     s[0] = c
     s[1] = 0
-    sys.stdout.write(s)
+    sys.stdout.write(to_string(<bytes> s))
```

Comment:
I don't like this since you are converting `bytes` to `str` which `sys.stdout` immediately needs to convert back to `bytes`. In Python 3, you can use `sys.stdout.buffer` for the underlying stream which accepts `bytes`. So you could do something like:
```
try:
    stdout_bytes = sys.stdout.bytes
except AttributeError:
    stdout_bytes = sys.stdout
stdout_bytes.write(s)
```